src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ChipsService.java

```diff
@@ -22,7 +26,12 @@ public ChipsService(ChipsClient chipsClient) {
     public void sendObjection(String requestId, Objection objection) {
         ChipsRequest chipsRequest = new ChipsRequest(
                 objection.getId(),
-                objection.getCompanyNumber()
+                objection.getCompanyNumber(),
+                objection.getAttachments(),
+                objection.getId(),
```

Comment:
Having a TODO comment at this point to say pass in the reference number once available would be useful.

Reply:
Put a TODO here and linked the jira issues.